build.gradle.kts

```diff
@@ -22,61 +22,63 @@ repositories {
 kotlin {
     targets {
         jvm()
-        iosX64()
+        iosX64("native")
```

Comment:
Is there a way we can leave this without a label? It then feels weird down below referencing the "native" target, but really it's an iOS target

Reply:
I think we can create a separate native source set that isn't bound to a particular target and the continue using the dependsOn pattern below

Reply:
Sure, it makes sense